AppKit/CPTableView.j

```diff
@@ -5098,6 +5098,17 @@ Your delegate can implement this method to avoid subclassing the tableview to ad
     }
     else if (!_isViewBased && [aView isKindOfClass:[CPControl class]] && ![aView isKindOfClass:[CPTextField class]])
     {
+        // we are currently in an edit session but the user clicked on a non-text control
+        if (_editingColumn >= 0)
+        {
+            var fr = [[CPApp keyWindow] firstResponder];
+            // make sure the currently edited control is commited
+            [self _commitDataViewObjectValue:fr];
+            // resend the same event in next run loop iteration
+            [[CPRunLoop currentRunLoop] performSelector:@selector(sendEvent:) target:CPApp argument:[CPApp currentEvent] order:0 modes:[CPDefaultRunLoopMode]];
```

Comment:
If you want to perform all selectors currently trapped in the run loop (is sendEvent:  one of them ?) , you can do `[currentLoop performSelectors]`